<!-- EN-Revision: 10934 -->
<sect1 id="zend.exception.using">

    <title>Verwenden von Ausnahmen</title>

    <para>
        Alle Ausnahmen welche von Zend Framework Klassen geworfen werden
        sollten eine Ausnahme werfen welche von der Basis Klasse
        Zend_Exception abgeleitet ist.
    </para>

    <example id="zend.exception.using.example">
        <title>Beispiel f端r das Fangen einer Ausnahme</title>
        <programlisting role="php"><![CDATA[
try {
    Zend_Loader::loadClass('nonexistantclass');
} catch (Zend_Exception $e) {
    echo "Gefangene Ausnahme: " . get_class($e) . "\n";
    echo "Nachricht: " . $e->getMessage() . "\n";
    // anderer Code um den Fehler zu korrigieren.
}]]>
        </programlisting>
    </example>

    <para>
        In der Dokumentation der einzelnen Zend Framework Komponenten
        stehen detailiertere Informationen dar端ber welche Methoden
        eine Ausnahme werfen, die Gr端nde f端r Ausnahmen und welche
        Ausnahme-Klassen von Zend_Exception abgeleitet sind.
    </para>

</sect1>
<!--
vim:se ts=4 sw=4 et:
-->
